src/somnium/congomongo.clj

```diff
@@ -585,19 +589,23 @@ You should use fetch with :limit 1 instead."))); one? and sort should NEVER be c
        :return-new? -> if true, the updated document is returned,
                        otherwise the old document is returned
                        (or it would be lost forever)
-       :upsert?     -> do upsert (insert if document not present)"
+       :upsert?     -> do upsert (insert if document not present)
+       :max-time-ms -> set the maximum execution time for operations"
   {:arglists '([collection where update {:only nil :sort nil :remove? false
-                                         :return-new? false :upsert? false :from :clojure :as :clojure}])}
-  [coll where update & {:keys [only sort remove? return-new? upsert? from as]
+                                         :return-new? false :upsert? false :max-time-ms 0 :from :clojure :as :clojure}])}
```

Comment:
this defaults to `nil` on fetch, but `0` here, is that intentional?

Reply:
Yes, due to differences in how we use it further. We use `(when...` and mutate cursor in fetch but fetch-and-modify just uses straight up Java method.

Reply:
ah right, your other option would be a conditional and using different overloads, which isn't an improvement 👍